Services/CO.CDP.Organisation.Authority/Endpoint.cs

```diff
@@ -66,6 +66,14 @@ public static void UseIdentity(this WebApplication app)
                         }
                         return Results.Problem(statusCode: StatusCodes.Status400BadRequest, detail: "Invalid client secret");
 
+                    case GrantTypes.RefreshToken:
+                        (valid, urn) = await service.ValidateRefreshToken(request.RefreshToken);
+                        if (valid)
+                        {
+                            return Results.Ok(await service.CreateToken(urn!));
```

Comment:
Wouldn't it be better to check if `urn != null` in the above condition, rather than use `urn!`? This way, it would fallback to returning a problem response, rather than a runtime exception.